beringei/tools/CMakeLists.txt

```diff
@@ -56,3 +56,4 @@ target_link_libraries(
 )
 
 add_subdirectory(grafana_read_service)
+add_subdirectory(plain_text_service)
```

Comment:
You can go ahead and rename grafana_read_service. Lets keep all the integrations in one place.